mock-backend/src/controllers/productController.js

```diff
@@ -19,10 +20,14 @@ const getAllProducts = (req, res) => {
   const response = {
     content: slicedProducts,
     totalPages: Math.ceil(products.length / size),
-    totalElements: products.length
+    totalElements: products.length,
```

Comment:
remove `,` symbol here

Reply:
It was added by prettier, so maybe we should keep it in order to stick to our formatting rules

Reply:
> It was added by prettier, so maybe we should keep it in order to stick to our formatting rules

For mock backend side we don't have prettier. Maybe it's your own settings
<img width="453" alt="image" src="https://github.com/user-attachments/assets/d490cd07-de7b-4ea1-a9a9-eea856d55cc9">

Reply:
Removed comma. Hope we will add prettier also to backend side to avoid such issues